src/helix-ui/elements/HXFileTileElement.less

```diff
@@ -38,11 +38,11 @@
     padding: 0.5rem 0.75rem;
 
     &:hover {
-      color: @cyan-500;
+      color: var(--hxColor, @cyan-500);
```

Comment:
If I wanted to override this variable, I'd need to be able to define it directly to the `:hover` selector in the Shadow DOM to override it.

```css
/* hypothetical CSS selector (doesn't work) */
hx-file-tile >>> #hxFileTile:hover {
  --hxColor: magenta;
}
```

If we could write the previous CSS, the custom properties would be useless. 

So we should probably rename this property to `--hxColor--hover`.

Reply:
While `--hxColor` works to change the color for all shadow elements, this hook should allow changes to just the "X" dismiss button.